Makefile

```diff
@@ -2,7 +2,7 @@
 SHELL := /bin/bash
 .PHONY: help
 
-ts := $(shell date --utc +%FT%TZ)
+ts := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Comment:
You might like `date --iso-8601=seconds` or `date -u --iso-8601=seconds`

Reply:
Doesn't work on macOS. 😢 
```date --iso-8601=seconds
date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]```

Reply:
:disappointed: